lib/ansible/modules/apt.py

```diff
@@ -135,6 +135,15 @@
     type: bool
     default: 'no'
     version_added: "2.1"
+  no_remove:
+    description:
+      - 'Corresponds to the C(--no-remove) option for I(apt).'
+      - 'If C(yes), it is ensured that no packages will be removed or the task will fail.'
+      - 'C(no_remove) is only supported with state except I(absent)'
+    aliases: ['no-remove']
```

Comment:
The parameter name should be a valid Python name and cannot contain dashes.
```suggestion
```

Reply:
OK, thanks.